src/main/java/org/launchcode/liftoffrecipetracker/models/Category.java

```diff
@@ -0,0 +1,59 @@
+package org.launchcode.liftoffrecipetracker.models;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+
+
+@Entity
+public class Category  {
+
+    //Class Variables
+    @GeneratedValue
+    @Id
+    private int id;
+    // do we want to specify size for category names?
+    //@Size(min = 4, max = 25, message = 'Category name must be between 4 and 25 characters long')
+    //@NotBlank
+    private String name;
+
+//    Add this after Recipe class is completed/extended
+//    @OneToMany(mappedBy = "category")
+//    private final List<Recipe> recipes = new ArrayList<>();
+
+
+    //Constructors
+
+    public Category(String name) {
+        this.name = name;
+    }
+
+    public Category() {}
+
+    //Methods
+
+
+
+    //Getters and Setters
+
+
+    public int getId() {
+        return id;
+    }
+
+    public void setId(int id) {
```

Comment:
Because we're using the @GeneratedValue and @Id annotations we probably don't need a setter here! Wouldn't want someone accidentally changing the value of ids. Let Spring manage that for us!

Reply:
good catch!